### PR TITLE
es translation for project slogan

### PR DESCRIPTION
**Pull request by sftim**
Translate “Production-Grade Container Orchestration”

/language es
/kind cleanup

Hope this is OK to target _master_ branch.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -222,7 +222,7 @@ language_alternatives = ["en"]
 
 [languages.es]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
+description = "Orquestación de contenedores para producción"
```
Comment (electrocucaracha) [inline config.toml]: Well, I'm not sure if this a better suggestion
```suggestion
description = "Orquestación de contenedores para ambientes de producción"
```

Comment (sftim) [reply config.toml]: I took the text from https://k8s.io/es/

I can propose changing both, so that they are synchronised. Should I do that?

Comment (electrocucaracha) [reply config.toml]: Let me verify this with the rest of the team, maybe they decided to choose this because it's shorter.

Comment (raelga) [reply config.toml]: We're still discussing the best approach, we will have something decided in a couple of days.

```suggestion
description = "Orquestación de contenedores para entornos productivos"
```

Comment (sftim) [reply config.toml]: OK cool. I'm happy to commit whatever the ES localization team recommend!

**Comment by kbhawkey** [comment]
@sftim, @raelga 
Is this pull request still active?

**Comment by sftim** [comment]
I still see this as active

**Comment by electrocucaracha** [comment]
@raelga maybe we can approve this as it is and change that later

**Comment by raelga** [comment]
Sorry for the delay, I'm getting back to pace. 😿 

/lgtm
/approve

**Comment by kbhawkey** [comment]
Hey @sftim ,
I realize that a member of the `es` localization team approved this PR and that you are helping out, but as you know approving your own PR is discouraged.

**Pull request merged**